### PR TITLE
[sonos] Fix sub/surround controls

### PR DESCRIPTION
**Pull request by lolodomo**
Related to #9874

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@cpmeister : can I request exceptionally an ultra fast review & merge so that it could be included in the snapshot that will be build this night ?

File: bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java [hunk 1]
```diff
@@ -2028,15 +2028,15 @@ public void setRepeat(Command command) {
...
+        setEqualizerBooleanSetting(command, "SubEnable");
     }
 
     public void setSubwooferGain(Command command) {
         setEqualizerNumericSetting(command, "SubGain", getSubwooferGain(), MIN_SUBWOOFER_GAIN, MAX_SUBWOOFER_GAIN);
     }
 
     public void setSurround(Command command) {
-        setEqualizerBooleanSetting(command, "SurroundEnabled");
+        setEqualizerBooleanSetting(command, "SurroundEnable");
```
Comment (cpmeister) [inline bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java:2039]: Shouldn't this be the same as what is expected in the SonosXmlParser?

Comment (lolodomo) [reply bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java:2039]: No and that is the reason of this quick fix.

**Review by cpmeister — APPROVED** [review]
Well, good luck then.

**Pull request merged**